.travis.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       python: 3.7
 
 install:
+  # Enable Python 3.6.x, when python: 3.5 (because black is Py3.6+)
+  - |
+    if [ "${TRAVIS_PYTHON_VERSION}" == "3.5" ]; then
```

Comment:
Does this mean 3.5 will actually be 3.5 ?

Reply:
No, it'll also contain a "python3.6" interpreter but "python" and "python3" refer to the system provided Python 3.5.

Reply:
Ok so this is done to make black available to the environment is it ?

Reply:
Yup. Black is python3.6+, so it won't install on Python 3.5.

Reply:
Then how is it supposed to run locally ? The 3.5 virtual env needs 3.6 to be installed as well ?

Reply:
You need to have both python 3.5 and python 3.6 installed. Whatever virtualenv you have, since there is a python3.6 visible (it will be, even on the python3.5 venv), pre-commit will use that interpreter for the isolated environment for Black.